R/format_PET.R

```diff
@@ -0,0 +1,1200 @@
+    dplyr::select(-.data$FemalesRingSeries,
+                  -.data$FemalesRing,
+                  -.data$NoString,
+                  -.data$NoNestBox) %>%
+    #### Reorder columns
+    dplyr::select(.data$BreedingSeason,
+                  .data$Species,
+                  .data$PopID,
+                  everything()) %>%
+    dplyr::distinct()
+
+
+
+  #### BROOD DATA
+
+  message("Compiling brood information...")
+  Brood_data <- create_brood_PET(parmaj_data,
+                                 fichyp_data,
+                                 jyntor_data)
+
+
+  #### CAPTURE DATA
+
+  message("Compiling capture information...")
+  Capture_data <- create_capture_PET(parmaj_data,
+                                     fichyp_data,
+                                     jyntor_data)
+
+
+  #### INDIVIDUAL DATA
+
+  message("Compiling individual information...")
+  Individual_data <- create_individual_PET(Capture_data)
+
+
+  #### LOCATION DATA
+
+  message("Compiling location information...")
+  Location_data <- create_location_PET(parmaj_data,
+                                       fichyp_data,
+                                       jyntor_data)
+
+
+  #### FINAL ARRANGEMENTS
+  Capture_data <-
+    Capture_data %>%
+    dplyr::select(-.data$BroodID)
+
+
+  time <- difftime(Sys.time(), start_time, units = "sec")
+  message(paste0("All tables generated in ", round(time, 2), " seconds"))
+
+
+  #### EXPORT DATA
+
+  if(output_type == "csv"){
+    message("Saving .csv files...")
+    utils::write.csv(x = Brood_data, file = paste0(path, "\\Brood_data_PET.csv"), row.names = F)
+    utils::write.csv(x = Capture_data, file = paste0(path, "\\Capture_data_PET.csv"), row.names = F)
+    utils::write.csv(x = Individual_data, file = paste0(path, "\\Individual_data_PET.csv"), row.names = F)
+    utils::write.csv(x = Location_data, file = paste0(path, "\\Location_data_PET.csv"), row.names = F)
+    invisible(NULL)
+
+  }
+
+  if(output_type == "R"){
+    message("Returning R objects...")
+    return(list(Brood_data = Brood_data,
+                Capture_data = Capture_data,
+                Individual_data = Individual_data,
+                Location_data = Location_data))
+
+  }
+}
+
+
+#### --------------------------------------------------------------------------~
+#### FUNCTIONS
+#### --------------------------------------------------------------------------~
+
+#### BROOD DATA
+
+#' Create brood data table for the populations from Institute of Biology, Karelian Research Centre, Russian Academy of Sciences, Petrozavodsk, Russia.
+#'
+#' Create a brood data table in standard format for great tits, pied flycatchers and Wryneck in Institute of Biology, Russia.
+#' @param pet_data Data frame. Primary data from Institute of Biology, Russia.
+#' @return A data frame.
+
+
+create_brood_PET <- function(parmaj_data,
+                             fichyp_data,
+                             jyntor_data) {
```

Comment:
These arguments aren't documented which causes many warnings in the check. Remember to look at the output of `devtools::check()`. There's more info in the developer workflow now https://github.com/SPI-Birds/pipelines#devtoolscheck